plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -12,6 +12,18 @@ import ServiceConfigUtil from '../utils/ServiceConfigUtil';
 import ServiceConfigBaseSectionDisplay from './ServiceConfigBaseSectionDisplay';
 import {findNestedPropertyInObject} from '../../../../../src/js/utils/Util';
 
+const getNetworkType = (networkType, appDefinition) => {
```

Comment:
It's true that functions should not be used as a "fancier syntax", but have a reason for being there.
In this particular case I have to agree with @nLight , and a `function` approach would be better:

```js
function getNetworkType(networkType, appDefinition) {
```